src/components/LandingPage/LandingPageImg/LandingPageImg.js

```diff
@@ -0,0 +1,15 @@
+import image from "./bg-landingPage.png";
```

Comment:
First, the file size is huge. Secondly, the image has low resolution. It needs to be corrected.

Reply:
Problem with the size and resolution is solved.

Reply:
Unfortunately it is not. And still no responsiveness in this view.
<img width="808" alt="Zrzut ekranu 2022-01-22 o 00 07 36" src="https://user-images.githubusercontent.com/38866668/150611634-f79b36ae-ef95-49d5-afed-8f83205be986.png">